.azure/pipelines/jobs/steps/frontend_packages.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      artifactName: node_modules
+      targetPath: frontend/node_modules
+
+  - bash: |
+      cd frontend
+      chmod +x node_modules/.bin/*
+      chmod +x -R node_modules/puppeteer/.local-chromium
```

Comment:
```suggestion
```
This isn't needed anymore, as for e2e the whole node_modules is installed freshly.